src/App.css

```diff
@@ -1,39 +1,127 @@
+
 .App {
-  text-align: center;
+  display: grid;
+  grid-auto-rows: 539px;
+  padding: 1rem 0;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+body {
+  background-color: #e5e5e5;
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  grid-template-columns: 100%;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0.5rem;
+  max-width: 750px;
+  margin-left: auto;
+  margin-right: auto;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.container .calculator {
+  height: 65vh;
+  width: 400px;
+  background-color: var(--main-black);
+  border-radius: 0.5rem;
+  box-shadow: 0 5px 10px 0 rgba(0, 0, 0, 0.5);
+}
+
+/* global button style */
+button {
+  border: unset;
+  box-shadow: unset;
+  color: var(--default-text-color);
+  cursor: pointer;
```

Comment:
- [ ] Great job with your design :clap:. Kindly make sure your design matches the provided template 🙏 

Original Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206484773-7cfe9a13-22ac-413a-9387-76eb7a7b772a.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206484732-6bc931e1-9519-4123-8ea4-df6b64ebdcb8.png)